ui/components/ui/definition-list/definition-list.stories.js

```diff
@@ -58,15 +58,15 @@ export const WithTypographyControl = () => (
     gapSize={select('gapSize', SIZES, SIZES.SM)}
     termTypography={{
       variant: select('termTypography.variant', TYPOGRAPHY, TYPOGRAPHY.H6),
-      color: select('termTypography.color', COLORS, COLORS.BLACK),
+      color: select('termTypography.color', COLORS, COLORS.TEXT_DEFAULT),
     }}
     definitionTypography={{
       variant: select(
         'definitionTypography.variant',
         TYPOGRAPHY,
         TYPOGRAPHY.H6,
       ),
-      color: select('definitionTypography.color', COLORS, COLORS.BLACK),
+      color: select('definitionTypography.color', COLORS, COLORS.TEXT_DEFAULT),
     }}
   />
 );
```

Comment:
Have no preview for this because storybook is broken right now waiting for https://github.com/MetaMask/metamask-extension/pull/14059